### PR TITLE
Temporary low res forecast model

### LINKED ISSUE #42: The number of processors at the JJOBs should be variable
Currently, the # processors at the JJOBS submitting scripts is hardcoded but should be variables, defined by the template at the workflow.

### PR DESCRIPTION
**Pull request by guillaumevernieres**
This PR adds the oops driven mom6 for the forecast step. 
* Default resolution is now 360x210x75, 1440x1080x75 can be used by editing "default/places.yaml"
* "Checkpoint" and "forecast" are in the 3dvar jjob scripts. Both application will fail for the 1/4 deg model, I'm assuming this is OK since the intent is to run the 1/4 deg when the emc model is part of this work flow.
closes #42

### CODE REVIEW

File: jobs/JJOB_DA_PREP
Comment (flampouris) [inline jobs/JJOB_DA_PREP]: RUNCDATE instead of RUNDIR, for consistency

File: jobs/JJOB_DA_PREP
Comment (flampouris) [inline jobs/JJOB_DA_PREP]: RUNCDATE instead of RUNDIR, for consistency

File: workflow/cases/3dvar.yaml
Comment (flampouris) [inline workflow/cases/3dvar.yaml]: Let's keep it for 2days long.

**Review by flampouris — CHANGES_REQUESTED** [review]
Add the forecast.yaml to the soca-config.

**Comment by guillaumevernieres** [comment]
> Add the forecast.yaml to the soca-config.

It should be in the soca-config, maybe pull again?

**Comment by flampouris** [comment]
I don't see it: https://github.com/JCSDA/soca-config/tree/develop/emc

**Comment by guillaumevernieres** [comment]
> I don't see it: https://github.com/JCSDA/soca-config/tree/develop/emc

readme points to feature/emc-3dvar, not in develop yet.

File: workflow/cases/3dvar.yaml
Comment (guillaumevernieres) [reply workflow/cases/3dvar.yaml]: sorry, got carried away!

**Comment by flampouris** [comment]
😬 my bad

**Comment by guillaumevernieres** [comment]
Just pushed the changes

**Review by flampouris — APPROVED** [review]
Tested for two days and works. But adding the model is a significant change, so let's wait till tomorrow for merging, to have the second review from @JessicaMeixner-NOAA.

Also, we have to communicate the change to all the users.

**Review by JessicaMeixner-NOAA — APPROVED** [review]

**Pull request merged**
